nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -124,7 +124,13 @@ in
       description = "Firefox Sync Server";
       wantedBy = [ "multi-user.target" ];
       path = [ pkgs.pythonPackages.pasteScript pkgs.coreutils ];
-      environment.PYTHONPATH = "${pkgs.pythonPackages.syncserver}/lib/${pkgs.pythonPackages.python.libPrefix}/site-packages";
+      environment.PYTHONPATH = let
+          penv = pkgs.python.buildEnv.override {
+            extraLibs = [ pkgs.pythonPackages.syncserver ];
+            ignoreCollisions = true;
+          };
+        in "${penv}/${pkgs.python.sitePackages}/";
```

Comment:
I get: collision between `/nix/store/61n8n63h906yqdqnyfdp6ifvppvvagrz-python2.7-WSGIProxy2-0.4.2/lib/python2.7/site-packages/wsgiproxy/__init__.pyc` and `/nix/store/sha7fmlklzxqq1fr7rhc3v9fhp98j6p8-python2.7-WSGIProxy-0.2.2/lib/python2.7/site-packages/wsgiproxy/__init__.pyc`